arch/x86/intel64.cmake

```diff
@@ -3,4 +3,9 @@
 
 zephyr_cc_option(-m64)
 
+set_property(GLOBAL PROPERTY PROPERTY_OUTPUT_ARCH "i386")
```

Comment:
Shouldn't it be `"i386:x86-64"` ? That is how it is defined in `linker-tool-gcc.h` that will be include in the linker scripts. That is really a question, I don't know how it really works on CMake and don't know which one has precedence.

Reply:
so far this works but I would prefer a centralized definition, will see if I can just use whats in the heade to make this simpler.

Reply:
I can't include C headers in CMake space.
It really doesn't matter in practice AFAICT. I set to "i386:x86_64" and it works the same.